omni-relayer/src/workers/near.rs

```diff
@@ -178,7 +183,24 @@ pub async fn claim_fee(redis_client: redis::Client, connector: Arc<OmniConnector
         let mut handlers = Vec::new();
 
         for (key, event) in events {
-            if let Ok(deposit_log) = serde_json::from_str::<Vec<u8>>(&event) {
+            if let Ok((block_number, deposit_log)) = serde_json::from_str::<(u64, Vec<u8>)>(&event)
+            {
+                let Ok(light_client_latest_block_number) =
+                    utils::near::get_eth_light_client_last_block_number(&config, &jsonrpc_client)
+                        .await
+                else {
+                    warn!("Failed to get eth light client last block number");
+                    continue;
+                };
+
+                if block_number > light_client_latest_block_number {
+                    tokio::time::sleep(tokio::time::Duration::from_secs(
+                        utils::redis::SLEEP_TIME_AFTER_EVENTS_PROCESS_SECS,
+                    ))
+                    .await;
+                    continue;
+                }
```

Comment:
In this function, we sometimes can use a wormhole, but we don't.

Reply:
Fixed in 1a8def1c8769bde5bacf888685ea0f803e717cef